### PR TITLE
Updated Terms of Service

### PR DESCRIPTION
**Pull request by thisandagain**
Resolves mozilla/webmaker-launch#90

### CODE REVIEW

File: src/pages/legal/legal.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+          <h2>8. Copyright and Trademark Notices</h2>
+          <p>To report copyright or trademark infringement, please visit <a target="_blank" href="https://www.mozilla.org/en-US/about/legal/report-abuse/">https://www.mozilla.org/en-US/about/legal/report-abuse/</a>.</p>
+
+          <h2>9. Termination</h2>
+          <p>The Terms will continue to apply until terminated by either you or Mozilla.</p>
+          <p>You may end your agreement with Mozilla regarding the Sites at any time for any reason by discontinuing your use of the sites and, if applicable, deactivating your account.If you stop using the Sites without deactivating your account, your account may be deactivated due to prolonged inactivity.</p>
```
Comment (k88hudson) [inline src/pages/legal/legal.jsx]: missing a space after "deactivating your account"

File: src/pages/legal/legal.less [hunk 2]
```diff
@@ -1,32 +1,34 @@
 #splash {
   .legal {
     display: block !important;
+    margin: 40px auto;
```
Comment (k88hudson) [inline src/pages/legal/legal.less:6]: you might want to add `padding: 0 20px` or something for mobile browsers so the text doesn't go right up against the edge

**Comment by k88hudson** [comment]
A couple of extremely minor comments feel free to skip if you dont have time. R+

File: src/pages/legal/legal.jsx (hunk 1 above)
Comment (k88hudson) [inline src/pages/legal/legal.jsx]: can you believe actually read through this lol

File: src/pages/legal/legal.less (hunk 2 above)
Comment (thisandagain) [inline src/pages/legal/legal.less:6]: :+1:

**Pull request merged**